dask/dataframe/io/tests/test_parquet.py

```diff
@@ -513,3 +513,26 @@ def test_timestamp96(fn):
     assert pf._schema[1].type == fastparquet.parquet_thrift.Type.INT96
     out = dd.read_parquet(fn).compute()
     assert_eq(out, df)
+
+
+def test_drill_scheme(fn):
+    N = 5
+    df1 = pd.DataFrame({c: np.random.random(N)
+                        for i, c in enumerate(['a', 'b', 'c'])})
+    df2 = pd.DataFrame({c: np.random.random(N)
+                        for i, c in enumerate(['a', 'b', 'c'])})
+    files = []
+    for d in ['test_data1', 'test_data2']:
+        dn = os.path.join(fn, d)
+        if not os.path.exists(dn):
+            os.mkdir(dn)
+        files.append(os.path.join(dn, 'data1.parq'))
+
+    fastparquet.write(files[0], df1)
+    fastparquet.write(files[1], df2)
+
+    df = dd.read_parquet(files)
+    assert 'dir0' in df.columns
```

Comment:
@mrocklin , `dir0 in df` doesn't seem to work, is this a bug? That would be a correct phrasing for pandas.

Reply:
I'm not sure where `'dir0` is coming from.  I would expect column names to be 'a', 'b', and 'c'.

You might also want to specify `index=False` to avoid the possibility of a sorted column being selected as the index automagically

Reply:
`dir0` is from the directory structure: the data files are one level deep, and the directory names are used as labels. This corresponds with the output of Drill, unlike Hive/Spark, which makes directory names like key=val, so we know what to call the columns.